src/ray/core_worker/store_provider/memory_store/memory_store.h

```diff
@@ -0,0 +1,60 @@
+#ifndef RAY_CORE_WORKER_MEMORY_STORE_H
+#define RAY_CORE_WORKER_MEMORY_STORE_H
+
+#include "ray/common/id.h"
+#include "ray/common/status.h"
+#include "ray/core_worker/common.h"
+#include "ray/core_worker/store_provider/store_provider.h"
+
+namespace ray {
+
+class GetRequest;
+class CoreWorkerMemoryStore;
+
+/// The class provides implementations for local process memory store.
+/// An example usage for this is to retrieve the returned objects from direct
+/// actor call (see direct_actor_transport.cc).
+class CoreWorkerMemoryStore {
+ public:
+  CoreWorkerMemoryStore();
+  ~CoreWorkerMemoryStore(){};
+
+  /// Put an object with specified ID into object store.
+  ///
+  /// \param[in] object The ray object.
+  /// \param[in] object_id Object ID specified by user.
+  /// \return Status.
+  Status Put(const RayObject &object, const ObjectID &object_id);
```

Comment:
The arguments here should be reversed, to make it more consistent with Get

Reply:
ok.